test-integ/peering_commontopo/README.md

```diff
@@ -1,12 +1,66 @@
-# peering_commontopo
+# CONSUL PEERING COMMON TOPOLOGY TESTS
```

Comment:
We can link this doc from https://github.com/hashicorp/consul/tree/main/docs#other-docs